### PR TITLE
Bridge front firelock tweak

### PR DESCRIPTION
**Pull request by Cocacolagua**
## What Does This PR Do
Moves the firelocks in front of the bridge a tiny bit to make it symmetrical.

## Why It's Good For The Game
Once you see it, you can never unsee it.

## Images of changes
Before
9YKjlKieBG

After
6JQGUnFC0K

## Changelog
:cl:
tweak: The firelocks in front of the bridge are now symmetrical between eachother.
/:cl:

### CODE REVIEW

**Comment by Qwertytoforty** [comment]
Thank you. This has bothered me for ages.

**Comment by SabreML** [comment]
It's probably important to note that this puts the firelocks into the 'Central Primary Hallway North' area, so they'll get triggered by the north fire alarm rather than the north east one.

*(This one)*
NSS Cyberiad 2021-03-19 155150

*(Rather than this one)*
NSS Cyberiad 2021-03-19 160412

**Review by Kyep — APPROVED** [review]

**Review by dearmochi — APPROVED** [review]

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -30227,15 +30224,18 @@
...
 /area/hallway/primary/central/ne)
 "bno" = (
-/obj/machinery/door/firedoor,
-/obj/machinery/atmospherics/pipe/simple/hidden/supply{
-	dir = 4
+/obj/structure/cable{
+	d1 = 4;
+	d2 = 8;
+	icon_state = "4-8";
+	tag = ""
```
Comment (S34NW) [inline _maps/map_files/cyberiad/cyberiad.dmm:30231]:
```suggestion
```

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 2]
```diff
@@ -30227,15 +30224,18 @@
 /turf/simulated/floor/plasteel,
 /area/hallway/primary/central/ne)
 "bno" = (
-/obj/machinery/door/firedoor,
-/obj/machinery/atmospherics/pipe/simple/hidden/supply{
-	dir = 4
+/obj/structure/cable{
+	d1 = 4;
+	d2 = 8;
+	icon_state = "4-8";
```
Comment (S34NW) [inline _maps/map_files/cyberiad/cyberiad.dmm:30230]:
```suggestion
	icon_state = "4-8"
```

**Review by S34NW — CHANGES_REQUESTED** [review]
please test this before committing, but this tag could be removed.

**Review by S34NW — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**